### PR TITLE
SAML support fixes

### PR DESCRIPTION
**Pull request by jfdenise**
Fix for #373 and #374

### CODE REVIEW

File: jboss/container/wildfly/launch/keycloak/2.0/module.yaml [hunk 1]
```diff
@@ -7,46 +7,40 @@ execute:
...
-  - name: "SSO_PRINCIPAL_ATTRIBUTE"
-    description: Principal attribute.
-  - name: "SSO_PUBLIC_KEY"
-    description: SSO Public Key.
+    description: The password of a user that has the priviledges to interact with the keycloak realm and can create and register clients.
   - name: "SSO_REALM"
     description: The SSO realm to which the application client(s) should be associated (e.g. demo).
   - name: "SSO_SAML_CERTIFICATE_NAME"
-    description: Certificate name
+    description: Alias of the private key in the SAML client keystore.
```
Comment (tommaso-borgato) [inline jboss/container/wildfly/launch/keycloak/2.0/module.yaml:22]: isn't it the alias of the certificate (the one EAP needs to store into RH-SSO at registration time)?

File: jboss/container/wildfly/launch/keycloak/2.0/module.yaml [hunk 2]
```diff
@@ -7,46 +7,40 @@ execute:
...
   - name: "SSO_TRUSTSTORE_CERTIFICATE_ALIAS"
-    description: Certificate alias.
+    description: Optional. Certificate alias inside the truststore.
   - name: "SSO_TRUSTSTORE_DIR"
-    description: Location of truststore.
+    description: Optional. Directory in which the truststore is located.
   - name: "SSO_TRUSTSTORE_PASSWORD"
-    description: The password for the truststore and certificate (e.g. mykeystorepass).
+    description: Optional. The password for the truststore and certificate (e.g. mykeystorepass).
   - name: "SSO_URL"
```
Comment (tommaso-borgato) [inline jboss/container/wildfly/launch/keycloak/2.0/module.yaml:43]: I had to add the suffix `/auth` for the SAML setup, perhaps we can add this info here

**Review by tommaso-borgato — COMMENTED** [review]
LGTM, I just a dded a few comments

File: jboss/container/wildfly/launch/keycloak/2.0/module.yaml (hunk 2 above)
Comment (jfdenise) [reply jboss/container/wildfly/launch/keycloak/2.0/module.yaml:43]: Thank-you, added.

**Pull request merged**